### ISSUE TITLE
Not reproducible

### ISSUE THREAD

**Issue by jrick** [jrick/release#1]
It's expected that with Go 1.13 and -trimpath this tool would produce reproducible builds.  This is not the case.  The following is diff'd hexdump output of dcrctl 1.4.0 for linux/amd64 built from an openbsd and darwin host:

```
--- dcrctl.h    Wed Sep  4 21:43:00 2019
+++ macos-dcrctl.h      Wed Sep  4 21:43:06 2019
@@ -128,10 +128,10 @@
 0000800    0000    0000    0000    0000    0000    0000    0000    0000
 *
 0000f90    0000    0000    0000    0000    0000    0000    0004    0000
-0000fa0    0053    0000    0004    0000    6f47    0000    4e74    472d
-0000fb0    4b70    4e51    566e    3941    384d    3754    4647    3964
-0000fc0    662f    484c    3332    5f4c    746a    4937    696d    3339
-0000fd0    5853    7457    2f4b    594f    4b51    4d76    4342    4769
+0000fa0    0053    0000    0004    0000    6f47    0000    5276    4739
+0000fb0    4f41    527a    6472    4c4e    5145    6944    5f6e    6e47
+0000fc0    5f2f    4a35    6966    5871    7534    5851    7462    4b41
+0000fd0    4755    4971    2f69    594f    4b51    4d76    4342    4769
 0000fe0    5a5f    5858    7333    3653    456b    5f2f    424f    6d5a
 0000ff0    6167    6d71    7049    7847    3143    6d33    5433    004b
 0001000    4864    0c8b    f825    ffff    48ff    613b    7610    4835
```

This needs further investigation.

**Comment by jrick** [comment]
SHA256 of each binary being inspected:

```
$ sha256 {,macos-}dcrctl                                                                                                                                                                      
SHA256 (dcrctl) = 5eca89601f4a98485a431c6bc04dde455a04a47f8a9cf1b6fd665d0edb8aa0e8
SHA256 (macos-dcrctl) = a966e579c7f7b02085802ab631471ffae21b6911d1eca81ce8cb14895cb283d8
```

**Comment by jrick** [comment]
https://github.com/golang/go/issues/33772

**Comment by jrick** [comment]
Fixed in dc96af5.
